Automattic/crowdsignal-ui
# New: Create Single Checkbox

# Summary

The PR adds a single checkbox as a new Form Block.

Depends on [D87147](https://code.a8c.com/D87147)

# Testing Instructions

- Apply the backend changes
- Open a new project and add a checkbox component
- Preview the survey and remove the `?preview=true` from the preview URL to log a record of the survey
- View the results in the app.crowdsignal.com dashboard to make sure both checked and unchecked responses are recorded.

## Suggestions for feedback

We created a block option to control the text of what's displayed on the results page. Any suggestions on the wording is appreciated.

![single-checkbox](https://user-images.githubusercontent.com/2293077/189733674-16d0c214-7bca-487b-9850-0117c271e5bc.png)
